BLUESPAWN-client/src/hunt/hunts/HuntT1035.cpp

```diff
@@ -99,7 +100,7 @@ namespace Hunts {
 	std::vector<std::shared_ptr<Event>> HuntT1035::GetMonitoringEvents() {
 		std::vector<std::shared_ptr<Event>> events;
 
-		ADD_ALL_VECTOR(events, Registry::GetRegistryEvents(HKEY_LOCAL_MACHINE, L"SYSTEM\\CurrentControlSet\\Services", false, false));
+		ADD_ALL_VECTOR(events, Registry::GetRegistryEvents(HKEY_LOCAL_MACHINE, L"SYSTEM\\CurrentControlSet\\Services", false, false, true));

```

Comment:
I would be very hesitant with this. A number of services somewhat regularly change information under their key, and since this hunt takes a long time to run, this is potentially dangerous. Can we instead also listen on the Parameters key of existing services?